playbooks/build-ansible-collection/run.yaml

```diff
@@ -10,6 +10,13 @@
         tox_install_siblings: false
         zuul_work_dir: "{{ zuul.projects['github.com/ansible-network/releases'].src_dir }}"
 
+    - name: Run tox environment(s) top repare the collection
```

Comment:
This could be dangerous and inject code at build time, is that something we'd like to do?  For most collections I don't think we want this, but if something you are comfortable on your collection, I feel we should have a dedicated build tarball jobs.

Rather then this, the job could parent to unittests job, which adds support for tools/test-setup.sh support.

Reply:
Indeed, from a security perspective, this can be a bit problematical.